from_cpython/Objects/listobject.c

```diff
@@ -109,6 +109,9 @@ PyList_Fini(void)
     }
 }
 
+static PyObject *indexerr = NULL;
+// Pyston change: comment this out
+#if 0
```

Comment:
It looks like there are now a lot of functions enabled in this file which we do not use.
I would really prefer if you just put a giant `#if 0` around the file and only enable the list sorting part